README.md

```diff
@@ -33,6 +33,21 @@ Within a `sct:packageDependency` element
 - **sct:editionDependency** states any necessary edition dependency and version using the SNOMED CT URI standard, e.g. `http://snomed.info/sct/900000000000207008/version/20220731`. Where a package is transitively dependent on multiple editions, only the direct non-transitive dependencies should be stated. Downloading packages would complete when there are no further dependencies.
 - **sct:derivativeDependency** states any necessary derivative dependency, as they are often published outside of SNOMED CT editions. Similarly, this should also follow the SNOMED CT URI standard for module and version
 
+### Naming conventions
+To aid human readability and distinguishing between packages, the following naming conventions should be applied.
+1. Edition packages
+     1. Should contain the word edition
+     2. Should NOT contain the word extension
+2. Extension packages
+     1. Should contain the word extension
+     2. Should NOT contain the word edition
+3. The version of the package should be included in the title
+4. The [RF2 Release Type](https://confluence.ihtsdotools.org/display/DOCRELFMT/3.2+Release+Types) Full/Delta/Snapshot should be included as a bracketed suffix to the title
```

Comment:
Are we discouraging the use of packages that contain multiple release types, for example a Snapshot and a Full? The International Edition package currently has this. 
I would be in favour of splitting this package and publishing two separate ones rather than having a package name that contains "Snapshot and Full".

Reply:
I'd support that too.

The Australian Edition was published as a Full and Snapshot and Delta package for a long time. In recognition that forced people to download more than they wanted when we did the NCTS we split it into 4 packages
- Full
- Snapshot
- Delta
- "All"

"All" is Full and Snapshot and Delta in the original style of all in one package for backward compatibility - had we not originally done that we probably wouldn't have continued it. I'd have to look at statistics of how many people actually download that package, but I think separate packages are better.